.gitignore

```diff
@@ -113,4 +113,12 @@ dist
 
 bundle/
 saucectl/
-screenshots/
\ No newline at end of file
+screenshots/
+
+# JS files are autogenerated.
+/src/**/*.js
+
+# Typescript mapping files are autogenerated.
+/src/**/*.d.ts
+/src/**/*.d.ts.map
```

Comment:
Lets check during standup whether there's a preference as to where to generate js output files. We have some projects that do it where the source file is, others output to a dedicated folder.